client/fixtures/subscription.ts

```diff
@@ -115,6 +115,60 @@ export const guardianWeeklySubscriptionAustralia: Subscription = {
 	deliveryAddressChangeEffectiveDate: '2021-12-24',
 };
 
+export const guardianWeeklySubscriptionUnitedStates: Subscription = {
+	paymentMethod: 'Card',
+	card: {
+		last4: '4242',
+		expiry: {
+			month: 3,
+			year: 2033,
+		},
+		type: 'Visa',
+		stripePublicKeyForUpdate: 'pk_test_123',
```

Comment:
I haven't traced the code fully, but it seems like the public key is coming from MDAPI in some? cases and it would be based off the billing country. https://github.com/guardian/members-data-api/blob/7094e7fd2927c97101fa11c7b29050aa55afd31b/membership-attribute-service/app/services/AccountDetailsFromZuora.scala#L52

If it's all tested with GW and other products and works ok then it's probably ok, I think GW is the most tricky with the billing vs delivery country plus currencies. (we lock the delivery country and currency together IIRC so that people definitely pay the right price)